src/proof/cnf_proof.cpp

```diff
@@ -349,6 +349,40 @@ void CnfProof::collectAssertionsForClauses(const IdToSatClause& clauses,
   }
 }
 
+// Detects whether a clause has x v ~x for some x
+// If so, returns the positive occurence's idx first, then the negative's
+Maybe<std::pair<unsigned, unsigned>> CnfProof::detectTrivialTautology(
```

Comment:
Use `size_t` for indices. `unsigned` is likely to be only 32-bit, even on 64-bit platforms.

Reply:
:+1: